amd_openvx/openvx/hipvx/CMakeLists.txt

```diff
@@ -21,27 +21,31 @@
 cmake_minimum_required(VERSION 3.5)
 project(openvx_hip CXX)
 
-set(HIPVX_SOURCES
+include_directories(${PROJECT_SOURCE_DIR}/../include ${PROJECT_SOURCE_DIR}/../ago ${PROJECT_SOURCE_DIR}/../api)
```

Comment:
why do we need this line added here?

Reply:
@asalmanp this folder is added when the HIP is detected as the backend, it does not have any deps with OpenVX but requires headers to compile